publisher/publisher_test.go

```diff
@@ -198,23 +118,43 @@ func errorBodyLogSrv() *httptest.Server {
 	return server
 }
 
-func setup(t *testing.T) (*Impl, *x509.Certificate, *ecdsa.PrivateKey) {
-	intermediatePEM, _ := pem.Decode([]byte(testIntermediate))
-
-	pub := New(nil,
+func setup(t *testing.T) (*Impl, []*x509.Certificate, *ecdsa.PrivateKey) {
```

Comment:
Any reason you changed this to return a slice of EE certs instead of a singular one? I don't see that functionality being taken advantage of anywhere.

Reply:
You're right, this is an artifact of a previous iteration that included the contents of a PEM bundle in the map. This can go.